### DISCUSSION TITLE
Currency field format method does not exist

### DISCUSSION THREAD

**Discussion by Baspa** [laravel/nova-issues#5398] [Q&A] [answered]
- Laravel Version: 10.0
- Nova Version: 4.0
- PHP Version: 8.1
- Reproduction Repository: Not available yet, but if needed I am willing to make one. 

### Description:

In the documentation is stated that you can format the currency field by using the `format()` method. However, using this results in a `Method Laravel\Nova\Fields\Currency::format does not exist.` error. 

I tried using the `formatMoney` method which can be found inside the Currency class but that gives me another error: `Expected type 'object'. Found 'string'` which I wouldn't expect because the `value` parameter is type hinted as `mixed`.

### Detailed steps to reproduce the issue on a fresh Nova installation:

1. Create a resource
2. Use the `Currency` field with the `format()` method: 

```php
Currency::make('Price', 'price')
    ->currency('EUR')
    ->format('%.2n')
    ->rules('required', 'numeric'),
```

**Comment by crynobone** [comment, marked as answer]
Only Laravel Nova v1 contains `format()` method.
